src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -208,7 +208,8 @@ def create(input_dataset, target, feature=None, validation_set='auto',
             + validation_set_corrective_string)
 
     dataset = _drop_missing_values(dataset, feature, is_train =True)
-    validation_dataset = _drop_missing_values(validation_dataset, feature, is_train=False)
+    if len(validation_set) > 0:
```

Comment:
Note: Travis is failing with
```
>       if len(validation_set) > 0:
E       TypeError: object of type 'NoneType' has no len()
```

Reply:
Per Shreya, this should check `len(validation_dataset)` not `len(validation_set)`. The former is guaranteed to be an SFrame. The latter can be a string or `None`.

Reply:
Looks like I forgot to push my last commit. Pushing now. Thanks for the catch!